staging/src/k8s.io/apiserver/pkg/apis/apiserver/types.go

```diff
@@ -62,8 +62,7 @@ type EgressSelectorConfiguration struct {
 // EgressSelection provides the configuration for a single egress selection client.
 type EgressSelection struct {
 	// Name is the name of the egress selection.
-	// Currently supported values are "controlplane", "master", "etcd" and "cluster"
-	// The "master" egress selector is deprecated in favor of "controlplane"
+	// Currently supported values are "controlplane", "etcd" and "cluster"
 	Name string
```

Comment:
custom Convert_v1alpha1_EgressSelection_To_apiserver_EgressSelection and Convert_v1beta1_EgressSelection_To_apiserver_EgressSelection implementations are needed to turn external `master` strings into `controlplane` internally

Reply:
Updated.